### PR TITLE
Untangled duplicate Tirana records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/983

- Deprecated, added is_current=0, and superseded one Tirana record into the other, added superseded_by to the current record.
- Brought over all extra `name:*`, `qs:*`, and `gn:*` properties
- Brought over all concordances
- Used the `wof:concordances_alt` property for the second `qs` concordance id

Kept the Tirana record beginning with `421`, deprecated the record beginning with `890` due to work done here: https://github.com/whosonfirst-data/whosonfirst-data/pull/735/files

### CODE REVIEW

File: data/421/187/037/421187037.geojson [hunk 1]
```diff
@@ -9,9 +9,13 @@
     "geom:bbox":"19.818599,41.3274,19.818599,41.3274",
     "geom:latitude":41.3274,
     "geom:longitude":19.818599,
+    "gn:country":"AL",
+    "gn:fcode":"PPLC",
+    "gn:pop":"374801",
     "iso:country":"AL",
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
```
Comment (nvkelso) [inline data/421/187/037/421187037.geojson]: Might as well mark this 1

File: data/890/444/545/890444545.geojson [hunk 2]
```diff
@@ -394,13 +397,15 @@
...
     "wof:id":890444545,
-    "wof:lastmodified":1501265906,
+    "wof:lastmodified":1507840812,
     "wof:name":"Tirana",
     "wof:parent_id":421182353,
     "wof:placetype":"locality",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
```
Comment (nvkelso) [inline data/890/444/545/890444545.geojson]: I think this one should be in `wof:superseded_by` instead.

Comment (stepps00) [reply data/890/444/545/890444545.geojson]: Fixed in 92fc6ea.

File: data/421/187/037/421187037.geojson (hunk 1 above)
Comment (stepps00) [reply data/421/187/037/421187037.geojson]: Fixed in 92fc6ea.

**Comment by stepps00** [comment]
This affects point geometries only, no descendant/PIP work is needed. 

Running through Boundary Issues.

**Comment by stepps00** [comment]
https://whosonfirst.mapzen.com/boundaryissues/pipeline/510/

**Pull request merged**